src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1101,6 +1101,21 @@
   <data name="DoNotUseUnsafeDllImportSearchPathMessage" xml:space="preserve">
     <value>Use of unsafe DllImportSearchPath value {0}</value>
   </data>
+  <data name="UseAutoValidateAntiforgeryToken" xml:space="preserve">
+    <value>Use antiforgery tokens in ASP.NET Core MVC controllers</value>
+  </data>
+  <data name="UseAutoValidateAntiforgeryTokenDescription" xml:space="preserve">
+    <value>Handling a POST, PUT, PATCH, or DELETE request without validating an antiforgery token may be vulnerable to cross-site request forgery attacks. A cross-site request forgery attack can send malicious requests from an authenticated user to your ASP.NET Core MVC controller.</value>
+  </data>
+  <data name="UseAutoValidateAntiforgeryTokenMessage" xml:space="preserve">
+    <value>Method {0} handles a {1} request without performing CSRF token validation</value>
```

Comment:
>Method {0} handles a {1} request without performing CSRF token validation [](start = 11, length = 73)

Method {0} handles a {1} request without performing antiforgery token validation. You also need to ensure that your HTML form sends an antiforgery token validation.

Reply:
Oops, shouldn't have "validation" at the end there.  I fixed and pushed to your repo.

---
In reply to: [315425704](https://github.com/dotnet/roslyn-analyzers/pull/2646#discussion_r315425704) [](ancestors = 315425704)